ignition/parameters/polygon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "ProxyModule": {
+    "name": "Denario Silver Coin",
+    "symbol": "DSC",
+    "minterAddress": "0x00",
```

Comment:
missing adresses

Reply:
yes, it is for production, so HQ has to add it upon deployment.